game/src/components/navbar.tsx

```diff
@@ -5,42 +5,59 @@ import { LogOut } from "lucide-react";
 import Image from "next/image";
 import Link from "next/link";
 import { Button } from "./ui/button";
+import { useEffect, useState } from "react";
+import { ethers } from "ethers";
```

Comment:
I don't think `ethers` is dep in `game`. You might have to use `viem` or install this.